SemDiff.Core/GitHub.cs

```diff
@@ -51,6 +52,25 @@ public GitHub(string repoOwner, string repoName, string authUsername = null, str
         public HttpClient Client { get; private set; }
         public string RepoFolder { get; set; }
         public string EtagNoChanges { get; set; }
+        public IList<PullRequest> currentSaved { get; set; }
+        public string JsonFileName { get; set; }
```

Comment:
This is also an option to allow the removal of the line in the constructor.
`public string JsonFileName { get; }  = "LocalList.json";`